docs/reference/glossary.asciidoc

```diff
@@ -61,6 +61,16 @@
   `object`. The mapping also allows you to define (amongst other things)
   how the value for a field should be analyzed.
 
+[[glossary-filter]] filter ::
+
+  A filter is kind of query known as a "non-scoring" query. It does not give a score,
+  it is only concerned about answering the question - "Does this document match?". 
+  The answer is always a simple, binary yes|no. This kind of query is said to be made 
+  in a "filtering" context, hence it is called a filter. Filtering queries/Filters are 
+  simple checks for set inclusion/exclusion. The goal of filtering is to reduce the 
+  number of documents that have to be examined, as it is in the case of 
+  <<glossary-query,scoring queries>>.
```

Comment:
I don't undestand the reference to scoring queries in this sentence, maybe you can explain or rephrase to make this clearer?

Reply:
The goal of filtering is to reduce the number of documents that have to be examined, **instead of what happens** in the case of  <<glossary-query,scoring queries>>.

The reference is also meant to provide a link to the reader for comparism